.gitignore

```diff
@@ -4,3 +4,140 @@
 /.idea/libraries
 .DS_Store
 /build
+
```

Comment:
I'm alright with the python standard .gitignore, but let's take out the Jetbrains .gitignores